docs/content/en/docs/getting_started.md

```diff
@@ -64,3 +64,45 @@ To run them manually, run
 ```bash
 pre-commit run --all-files
 ```
+
+## Creating a New Pass
+
+The `templates` folder contains Python scripts to create boilerplate for new conversion or (dialect-specific) transform passes.
```

Comment:
Maybe add `python templates/template.py --help` which should display a nice help string (or maybe we need to add that).

Reply:
Currently doesn't work, but I'll fix in a follow up issue!